pkg/state/helmx.go

```diff
@@ -26,6 +26,23 @@ func (st *HelmState) appendHelmXFlags(flags []string, release *ReleaseSpec) ([]s
 	return flags, nil
 }
 
+// append post-renderer flags to helm flags
+// nolint: unparam
+func (st *HelmState) appendPostRenderFlags(flags []string, release *ReleaseSpec, helm helmexec.Interface) ([]string, error) {
+	if helm.IsHelm3() {
+		switch {
+		// helm.GetPostRenderer() comes from cmd flag.
+		case helm.GetPostRenderer() != "":
```

Comment:
@mumoshu helm.GetPostRenderer() != "" will get postRenderer of HelmExec.

Reply:
Ah gotcha! Thanks for clarifying